monitoring/prober/scd/test_operation_simple_heavy_traffic_concurrent.py

```diff
@@ -11,8 +11,12 @@
   - confirm deletion by get and query
```

Comment:
If I try to convert `test_mutate_ops_concurrent` to async function, it starts failing at:
`op_req_map[op_id] = _build_mutate_request(idx, op_id, op_map, scd_session, scd_api)`

with following error: 
`All tests must specify auth scope for all session requests.  Either specify as an argument for each individual HTTP call, or decorate the test with @default_scope.`